app/web/components/GlobalMessage.tsx

```diff
@@ -1,13 +1,32 @@
 import { Alert as MuiAlert } from "@material-ui/lab/";
-import React, { useEffect, useState } from "react";
+import { usePersistedState } from "platform/usePersistedState";
+import React, { useEffect } from "react";
 
 interface GlobalMessageData {
+  // the unix timestamp (as a string with milliseconds) when the message was issued
+  epoch: string;
   severity: "success" | "info" | "warning" | "error";
   message: string;
 }
 
+const TIME_BETWEEN_CHECKS_MS = 300_000; // 5 min
```

Comment:
Should it have the underscore?

Reply:
It's a thousands separator to make it more readable: https://github.com/tc39/proposal-numeric-separator